src/helpers/privacy_budget.rs

```diff
@@ -0,0 +1,80 @@
+/// This module implements privacy budget using a redis cache
+/// It expects a vector of encrypted blobs alongwith budget to be consumed.
+/// After increasing the budget consumed by all the blobs, it returns blobs which are below ``TOTAL_PRIVACY_BUDGET``
+
+use redis::Commands;
+use std::env;
+
+const TOTAL_PRIVACY_BUDGET: i32 = 100;
```

Comment:
This might be better as a property on a struct (see other comments).  It also needs to be public  (with a getter if it is a property) so that the caller can know how much budget is available.